pkg/policy/policies_compute.go

```diff
@@ -89,18 +89,26 @@ func (ps *policies) calculateGlobalMinMax() {
 	for _, p := range ps.allFromMap() {
 		if p.UIDFilter.Enabled() {
 			if !uidMinFilterableInUserland && p.UIDFilter.Minimum() != filters.GetUnsetMin[uint32]() {
-				ps.uidFilterMin = common.Min(ps.uidFilterMin, uint64(p.UIDFilter.Minimum()))
+				if ps.uidFilterMin > uint64(p.UIDFilter.Minimum()) {
```

Comment:
These are removing the Min(). It was intended?

Reply:
Yes. mMin and Max were trivial functions used in only one place, so I prefered to remove them